### DISCUSSION TITLE
Doris在使用select into outfile 导出文件到本地的时候有没有办法指定导出到指定BE所在服务器

### DISCUSSION THREAD

**Discussion by liujiayin02** [apache/doris#24531] [A - General / Q&A] [answered]
工作需要获取Doris导出文件到本地，但是目前Doris是随机指定BE节点输出文件，因为要跨服务器拿取文件（不允许使用scp等命令，只能通过工单获取，但是这样就需要固定一个服务器）希望能固定在一台服务器上输出，但是我看官方文档没有相关配置。有没有办法实现导出文件在指定服务器上。

**Comment by ixzc** [comment, marked as answer]
Not supported yet.

**Reply by liujiayin02** [reply]
thanks
